spec/plainmonthday.html

```diff
@@ -487,15 +472,7 @@ <h1>CreateTemporalMonthDayFromStatic ( _constructor_, _month_, _day_, _reference
       <h1>ToTemporalMonthDayFields ( _temporalMonthDayLike_, _fieldNames_ )</h1>
       <emu-alg>
         1. Assert: Type(_temporalMonthDayLike_) is Object.
-        1. Let _result_ be ? OrdinaryObjectCreate(%Object.prototype%).
-        1. For each value _property_ of _fieldNames_, do
-          1. Let _value_ be ? Get(_temporalMonthDayLike_, _property_).
-          1. If _property_ is one of *"month"* or *"year"*, then
-            1. If _value_ is *undefined*, throw a *TypeError* exception.
-            1. Set _value_ to ? ToIntegerOrInfinity(_value_).
-          1. Perform ! CreateDataPropertyOrThrow(_result_, _property_, _value_).
-        1. Let _year_ be the first leap year after the Unix epoch (1972).
-        1. Perform ! CreateDataPropertyOrThrow(_result_, *"year"*, _year_).
+        1. Let _result_ be ? PrepareTemporalFields(_temporalMonthDayLike_, _fieldNames_, «»).
```

Comment:
```suggestion
        1. Let _result_ be ? PrepareTemporalFields(_temporalMonthDayLike_, _fieldNames_, « *"year"* »).
```
?

Reply:
Cannot require year on input bag because calendar may instead use era, eraYear; I thought we were delegating this to the calendar to throw if it couldn't produce a year and is expected to (i.e. not MonthDayFromFields). In cases where temporal wants to expect one of `{year, era&eraYear}`, should I extend this to have something like "if <<required>> includes 'year'"?

Reply:
oh, right, the List is of required fields, not optional fields. In that case, disregard what I said.